src/pages/Home.jsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { useState, useEffect } from 'react'
+import axios from 'axios'
+import '../Styles/Home.css'
+
+
+const Home = () => {
+    
+    
+    const [homeArticles, setHomeArticles] = useState([])
+
+
+    useEffect(() => {
+        axios.get('https://nc-news-fz7g.onrender.com/api/articles')
+        .then(({data}) => {
+        console.log(data, '<<< consoling the useEffect')
+        setHomeArticles(data.articles)
```

Comment:
No articles are being shown here, the console logs are all showing empty arrays. Perhaps check you are getting what is returned off the correct key 🔑